pheps/phep-9999.md

```diff
@@ -0,0 +1,112 @@
+   - pip Installable: indicates whether a package is pip installable
+   - conda Installable: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:
+
+| Tier                  | Summer School Inclusion | PyHC Software Env Inclusion | PyHC-Chat Bot Inclusion | pyOpenSci Verified badge | Standards Compliance Assistance | Listing on Main PyHC Project Page | Listing on Secondary PyHC Project Page | Software Search Interface Inclusion | Consideration for Conference Travel Funding |
+| :--:                  | :---------------------: | :-------------------------: | :---------------------: | :----------------------: | :-----------------------------: | :-------------------------------: | :------------------------------------: | :---------------------------------: | :-----------------------------------------: |
+| **Gold**              | Yes                     | Yes                         | Yes                     | Yes                      | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Bronze**            | No                      | No                          | No                      | No                       | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
+| **Silver**            | No                      | No                          | No                      | No                       | No                              | Yes                               | No                                     | Yes                                 | No                                          |
+| **Honorable Mention** | No                      | No                          | No                      | No                       | No                              | No                                | Yes                                    | Yes                                 | No                                          |
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC Software Env Inclusion: indicates whether a package will be included within the PyHC software environment
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
+   - pyOpenSci Verified Badge: a badge that shows whether a package has completed the pyOpenSci review process
+   - Standards Compliance Assistance: indicates whether a package will receive extra help and/or advice from PyHC leadership in conforming to the PyHC standards
```

Comment:
So, this seems counterintuitive to me.  Shouldn't the packages that are less up-to-snuff receive more help?  Perhaps a better way of splitting time would be having a pool of volunteers that could give time for things like code reviews and then receive code reviews from other people.  That would be a nice way of creating more of a community environment in PyHC, but is not directly related to the tiers.

Reply:
Fair point. There's also an idea that packages who've done the work to be at a higher level get more help if there's extra funding for someone to poke around their code base (e.g. work through bug issues or solve long-standing open PRs).

Reply:
That's a good point!  We should have community support mechanisms for packages to level up, as you suggest.  

📌 I'm wondering if something like an annual unstructured Helio Hack Week (like Astro Hack Week) would provide good opportunities for us to support each other in this way.

Reply:
As a potential use case, having a path forward to help packages be pip/conda-installable would be a good support. A Hack Week could be useful here.